bindata/network/multus/002-rbac.yaml

```diff
@@ -53,37 +54,153 @@ rules:
   - create
   - patch
   - update
+- apiGroups: ["certificates.k8s.io"]
+  resources:
+    - certificatesigningrequests
+  verbs:
+    - create
+    - get
+    - list
+    - watch
+
+---
+# This role is ancillary tools in the multus namespace.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: multus-ancillary-tools
+rules:
+- apiGroups: ["apiextensions.k8s.io"]
+  resources:
+  - customresourcedefinitions
+  - customresourcedefinitions/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["k8s.cni.cncf.io"]
+  resources: ["*"]
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: [""]
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: [""]
+  resources:
+  - nodes
+  verbs:
+  - get
+- apiGroups: ["", "events.k8s.io"]
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - update
 
+# We keep this service account for upgrades.
+# It will be unreferenced and unused after the upgrade.
+# We assign it a new ClusterRoleBinding to a less permissive ClusterRole
 ---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: multus
   namespace: openshift-multus
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: multus-transient
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: multus-ancillary-tools
+subjects:
+- kind: ServiceAccount
+  name: multus
+  namespace: openshift-multus
 
+
+{{ if .NETWORK_NODE_IDENTITY_ENABLE }}
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: multus-group
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: multus-proper
+subjects:
+  - kind: Group
+    name: system:multus
+    apiGroup: rbac.authorization.k8s.io
+{{ else }}
+# Network node identity is disabled, therefore, we use a service account for Multus itself
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: multus-proper
+  namespace: openshift-multus
```

Comment:
We don't need this here anymore, we will always have the SA

Reply:
got it. I can change the following clusterrolebinding to bind to the extant `multus` serviceaccount.